examples/node/getinfo.js

```diff
@@ -9,13 +9,11 @@
 
 var fs = require('fs');
 
-// HACK few hacks to let PDF.js be loaded not as a module in global space.
-global.window = global;
-global.navigator = { userAgent: "node" };
-global.PDFJS = {};
+// HACK adding DOMParser to read XMP metadata.
 global.DOMParser = require('./domparsermock.js').DOMParserMock;
 
-require('../../build/singlefile/build/pdf.combined.js');
+// Run `node make dist` to generate 'pdfjs-dist' npm package.
```

Comment:
Nit: let's keep this the same as in the other example file and add "files" after "package", i.e., make it "Run `node make dist` to generate 'pdfjs-dist' npm package files."